releasenotes/notes/Deprecate_legacy_pulse_builder_command-2a3c8dcbceae624e.yaml

```diff
@@ -0,0 +1,63 @@
+---
+deprecations:
+  - |
+    Deprecated injecting circuit gate operation into the pulse context.
+    
+    Deprecated
+    :func:`~qiskit.pulse.builder.call_gate`,
+    :func:`~qiskit.pulse.builder.cx`,
+    :func:`~qiskit.pulse.builder.u1`,
+    :func:`~qiskit.pulse.builder.u2`,
+    :func:`~qiskit.pulse.builder.u3`,
+    :func:`~qiskit.pulse.builder.x`
+
+    .. code-block:: python
+
+      """ Users are encouraged to use gates only in relm of circuits, NOT in pulse(s)
+        Pulse builder should exclusively be used with pulse schedules (like Drag, Gaussian, GaussianSqaure, etc.)
+        Example:"""
+      with pulse.build() as only_pulse_scheds:
+        pulse.play(Drag(180, 0.2, 40, -1.1), DriveChannel(0))
+        pulse.play(Gaussian(160, 0.14, 30), DriveChannel(1))
+        
+      """If a user wishes to get the pulse schedules for a gate.
+      This example shows how to get pulse gate instructions."""
+      from qiskit.providers.fake_provider import FakePerth
+      backend=FakePerth()
+      sched=backend.target['x'][(qubit,)].calibration
+
+      # For BackendV1, like FakeNairobi
+      from qiskit.providers.fake_provider import FakeNairobi
+      FakeNairobi().defaults().instruction_schedule_map.get('x', (qubit,))
+
+      """If a user wishes to get the pulse schedules of a QuantumCirucit"""
+      from math import pi
+      qc = QuatumCircuit(2)
+      qc.rz(pi/2, 0)
+      qc.sx(0)
+      qc.rz(pi/2, 0)
+      qc.cx(0, 1)
+      
+      from compiler import schedule, transpile
+      qc_t = transpile(qc, backend)
+      schedule(qc_t, backend)
```

Comment:
```suggestion

    If you still wish to inject backend calibrated schedules, 
    you can use following pattern instead of calling gate commands.

    .. code-block:: python

      from qiskit.providers.fake_provider import FakePerth
      from qiskit import pulse
      
      backend = FakePerth()
      sched = backend.target['x'][(qubit,)].calibration
      
      with pulse.build() as only_pulse_scheds:
          pulse.call(sched)
      
    Similarly, :class:`.QuantumCircuit` can be injected in the builder context
    by manually transpiling and scheduling the object.

    .. code-block:: python
                                                                                
      from math import pi
      from qiskit.compiler import schedule, transpile

      qc = QuatumCircuit(2)
      qc.rz(pi/2, 0)
      qc.sx(0)
      qc.rz(pi/2, 0)
      qc.cx(0, 1)
      
      qc_t = transpile(qc, backend)
      sched = schedule(qc_t, backend)

      with pulse.build() as only_pulse_scheds:
          pulse.call(sched)

    In any case we now recommend to write a minimum pulse program
    with the builder and attach it to :class:`.QuantumCircuit` via the
    :meth:`.QuantumCircuit.add_calibration` method as a microcode of a
    gate instruction, rather than writing the entire program with the pulse model.
```

I would write comment block outside the code block. I think having such migration guide is useful to users. Thanks!

Reply:
Applied suggestion